docs/guides/action_triggers.md

```diff
@@ -0,0 +1,208 @@
+---
+page_title: Retrieve triggers available within actions
+description: |-
+Retrieve the set of triggers currently available within actions.
+---
+
+# Retrieving the set of triggers available within actions
+
+In this guide we'll show how to retrieve the set of triggers currently available within actions.
+A trigger is an extensibility point to which actions can be bound.
+
+## Get an API Explorer Token
+
+Head to the APIs section of your [Auth0 Dashboard](https://manage.auth0.com/#/apis) and click on the 
+**Auth0 Management API**.
+
+<img alt="get_api_explorer_token_1" src="https://user-images.githubusercontent.com/28300158/196163108-1a871ae9-7d98-4c4b-bb9f-a1fbf2e3fe8f.png">
+
+Click on the **Create & Authorize Test Application** within the **API Explorer** tab.
```

Comment:
Our docs guidelines state we should prefer 'Select' to 'Click' because of screen readers and similar devices (same for other instances in this file).

Reply:
Thanks for spotting that @Widcket 🙇🏻 , I fixed it in https://github.com/auth0/terraform-provider-auth0/pull/370/commits/34d5daeaddf81ca7158d84059daa56eee2a773c2.